plugin/pkg/scheduler/schedulercache/cache_test.go

```diff
@@ -443,9 +459,10 @@ func TestRemovePod(t *testing.T) {
 
 func BenchmarkGetNodeNameToInfoMap1kNodes30kPods(b *testing.B) {
 	cache := setupCacheOf1kNodes30kPods(b)
+	nodeInfoMap := make(map[string]*NodeInfo)
 	b.ResetTimer()
 	for n := 0; n < b.N; n++ {
-		cache.GetNodeNameToInfoMap()
+		cache.UpdateNodeNameToInfoMap(nodeInfoMap)
```

Comment:
This benchmark doesn't make much sense now, becuase it's basically no-op.

However, I'm running 1000-node cluster experiments on kubemark. And before this change, this function was ~7-9% of the profile, after the change it was <1%. So the change is really huge...